src/plat-net/posix/nyoci-plat-net.c

```diff
@@ -617,11 +677,14 @@ sendtofrom(
 			pktinfo->ipi_addr = ((struct sockaddr_in*)saddr_from)->sin_addr;
 			pktinfo->ipi_ifindex = 0;
 		}
-
+        
 		ret = sendmsg(fd, &msg, flags);
 
 		check(ret > 0);
 		check_string(ret >= 0, strerror(errno));
+#else // !NYOCI_CAN_SENDMSG
+		abort(); //TODO
```

Comment:
It's a little hard to tell what is going on here from looking at the diff on GitHub. Why is this `abort()` here? Would an `#error` be more suitable?